src/DIRAC/Core/Tornado/Client/TornadoClient.py

```diff
@@ -73,11 +74,38 @@ def receiveFile(self, destFile, *args):
         :param args: list of arguments
         :returns: S_OK/S_ERROR
         """
-        rpcCall = {"method": "streamToClient", "args": encode(args)}
+        rpcCall = {
```

Comment:
I don't think this changes makes sense. There are plenty of cases where the arguments have nothing to do with `fileIDs`.

Reply:
Not sure I get your comment. This is the standard TransferClient receiveFile signature. fileID can be a complex structure itself, indeed.

Reply:
For example. DIPStorage has:
```
    def __getFile(self, src_url, dest_file):
        transferClient = TransferClient(self.url)
        res = transferClient.receiveFile(dest_file, src_url, token=self.checkSum)
        ...
```
which fails otherwise with unexpected key token

Reply:
OK, I have rolled back this change and fixed it in the DIPStorage instead.